package.json

```diff
@@ -43,14 +43,14 @@
     "prepublishOnly": "npm run lint && npm run test"
   },
   "devDependencies": {
-    "babel-jest": "^23.0.1",
+    "babel-jest": "^24.9.0",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
-    "eslint": "^4.19.1",
-    "eslint-config-ktsn": "^1.0.3",
-    "eslint-config-prettier": "^2.9.0",
-    "jest": "^23.1.0",
-    "prettier": "1.13.3",
-    "svelte": "^2.7.0"
+    "eslint": "^6.1.0",
+    "eslint-config-ktsn": "^2.0.1",
+    "eslint-config-prettier": "^6.0.0",
+    "jest": "^24.9.0",
+    "prettier": "1.18.2",
+    "svelte": "^3.8.1"
   },
   "peerDependencies": {
     "svelte": "*"
```

Comment:
Since this change is not compatible with old version, can you update the peerDep to `svelte@^3.0.0`?

Reply:
Yeah, I didn't notice that :+1: